tests/performance/framework.go

```diff
@@ -53,12 +56,27 @@ func init() {
 	}
 }
 
-func SIGDescribe(text string, body func()) bool {
-	return Describe("[sig-performance][Serial] "+text, body)
+func SIGDescribe(text string, body ...interface{}) bool {
+	if !isIncludeSerial(body...) {
+		body = append(body, Serial)
+	}
+	return Describe("[sig-performance][Serial] "+text, body...)
+}
+
+func FSIGDescribe(text string, body ...interface{}) bool {
+	if !isIncludeSerial(body...) {
+		body = append(body, Serial)
+	}
+	return FDescribe("[sig-performance][Serial] "+text, body...)
 }
 
-func FSIGDescribe(text string, body func()) bool {
-	return FDescribe("[sig-performance][Serial] "+text, body)
+func isIncludeSerial(body ...interface{}) bool {
```

Comment:
This is needed because gingko considers multiple `Serial` items in the slice as an error? If this is the case, I tend to think it is a bug.

Anyway, I would prefer to fix the duplication and be done with it, over doing this nasty hack.
Is it possible?

Reply:
This change is not directly related to this PR, but it's needed for this PR. The spec APIs were changed from v1 to v2, and so these custom envelopes (`SIGDescribe`, `FSIGDescribe`) need to be updated to match the API change of the `Describe` container. No hack here.

Reply:
The comment is on `isIncludeSerial` addition, which checks inclusiveness of the `Serial` item. I am not talking about the signature change.

Reply:
I peeked at the source-code of ginkgo. I do not really understand why we need this check at all.
It is suppose to be fine even if multiple identical `Serial` arguments are included in the args. What exactly did not work for you that you had to do this?

Reply:
OK, so I don't think ginkgo cares about duplication here. I can drop this code, but I'll check ginkgo source code first, to be sure.

Reply:
Fixed, thanks!